aiida/storage/sqlite_temp/backend.py

```diff
@@ -133,7 +133,7 @@ def get_repository(self) -> SandboxRepositoryBackend:
             raise ClosedStorage(str(self))
         if self._repo is None:
             # to-do this does not seem to be removing the folder on garbage collection?
-            self._repo = SandboxRepositoryBackend(sandbox_in_repo=True)
+            self._repo = SandboxRepositoryBackend()
```

Comment:
In all the other replacements I've seen you first do:

```python
filepath_sandbox = get_config_option('storage.sandbox') or None
```

And then provide that when creating the `SandboxRepositoryBackend` or the `SandboxFolder`, yet here it will always create the temporary folder using the temp location of the system. Was this intended? Not saying is wrong, just checking the difference, specially since here it is now changing the behavior.

Reply:
Good catch, I didn't make this decision consciously, I think I just missed it. Maybe for consistency it would be good to have the same behavior here and also use the option.